### DISCUSSION TITLE
Usable as Tile Server?

### DISCUSSION THREAD

**Discussion by kennethhaspeel** [SixLabors/ImageSharp.Web#274] [Q&A]
I was wondering if this project is suitable to use as a Tile Server for (example) OpenLayers based on xyz.
I'm looking to serve a 4GB jp2 image in several OpenLayers projects. I could use something like GeoServer but to set this up for serving just one Tile Layer looks like overkill.

Everything runs in a .Net Core 5.0 MVC project

I found [THIS](http://build-failed.blogspot.com/2016/06/creating-simple-tileserver-with-net.html) which is a good start. It is used to draw and serve as tile layer. But it references a (retired) project ImageProcessor by @JimBobSquarePants so i'm kinda assuming he is the best person to answer my question. 

What i'm trying to achieve:

- serve a jp2 in a way it can be used in Openlayers as ol.layer.Tile
- build a cache of the served images

It would take me a while to figure out how to do this. So before I start looking for hours, i just wanted to ask if it was even possible

Not sure if i need to give more information.

**Comment by JimBobSquarePants** [comment]
A quick scan of that article suggests You would probably implement your own [`IImageWebProcessor`](https://docs.sixlabors.com/api/ImageSharp.Web/SixLabors.ImageSharp.Web.Processors.IImageWebProcessor.html) to do it
